dso-l2/src/main/java/com/tc/objectserver/persistence/FlatFileStorageServiceProvider.java

```diff
@@ -84,5 +91,18 @@ public boolean initialize(ServiceProviderConfiguration configuration) {
   public void close() {
     
   }
+
+  @Override
+  public void clear() throws ServiceProviderCleanupException {
+    for(FlatFilePersistentStorage storage : storages) {
```

Comment:
I have a feeling that this will actually have to walk the underlying directory, deleting any files it finds, since we have not opened them all, yet.